### DISCUSSION TITLE
Jamku v4 Design 1st Look

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#814] [Announcements]
![image](https://user-images.githubusercontent.com/11911938/142807154-12315441-d2b3-4801-a0e4-bb056ace0327.png)
Jamku v4 Task List Design

### 1. Moving lengthy navigation menu to sub menu
This makes all the related reports and pages easy to access.
Depending on upon the Features activated, the Main Menu will no longer have extra items. Example: If you don't use attendance, it will be hidden from main menu.
![image](https://user-images.githubusercontent.com/11911938/142807337-9bb78cc1-e67c-4f31-a2df-f7e0377366a5.png)

### 2. Subtle Animations
For a delightful experience and to provide a sense of fluidity, there are subtle animations while interacting with software.
![TaskStatusAnimation](https://user-images.githubusercontent.com/11911938/142808028-77b7465c-c4ac-45bd-8a5e-1b4d703564d9.gif)

 
### 3. Visual Cues
Instead of providing text everywhere, we are using combination of Color, Images, Shadows 
![image](https://user-images.githubusercontent.com/11911938/142808537-8f3ac4cf-0a55-48cd-801b-bafd9ff63772.png)

### 4. Colourful
To get the user from the monotonous work to more cheerful mood, we selected a colourful design scheme.
![image](https://user-images.githubusercontent.com/11911938/142976294-a8a90d9a-0559-46d1-8c61-e0e555912947.png)

### 5. Better Filtering
Filtering is the most important aspect of the day to day working and reporting.
Hence, we have a dedicated pane for just that.
![image](https://user-images.githubusercontent.com/11911938/142976483-aeb0d27c-2b8b-42da-aebb-b020f8b70a83.png)

### 6. Select Column you need
Every user's needs are different. Rearrange and select the columns you need.
![image](https://user-images.githubusercontent.com/11911938/142976666-82f0b9aa-dbd2-4c8e-bb13-17191abbd8ee.png)

Design by [CA Manas Madrecha](https://manas.madrecha.com/)

**Comment by adarshmadrecha** [comment]
# Evolution of Task List Page

v1 Jamku (2013)
![image](https://user-images.githubusercontent.com/11911938/142976817-eea78a2d-43c8-452a-88ff-58b777c8869f.png)

v2 Jamku (2017)
![image](https://user-images.githubusercontent.com/11911938/142977121-5ef0d62c-614f-49b1-b395-54a8fc7da289.png)


v3 Jamku (2019)
![image](https://user-images.githubusercontent.com/11911938/142977287-cfd8c3ce-4497-4be4-9d7b-386d89d063d1.png)


v4 Jamku (2022 *planned)
![image](https://user-images.githubusercontent.com/11911938/142807154-12315441-d2b3-4801-a0e4-bb056ace0327.png)

**Reply by vatsalbhatia** [reply]
Pl ensure we continue to have tabular (excel) type of view in addition to whatever we choose to add. This is very useful as of now.

**Reply by adarshmadrecha** [reply]
@vatsalbhatia thanks for the feedback

**Comment by capostwal** [comment]
The Design looks very fresh and simplified 
Request you to please include Simplified method for billing as well.
Task #448

**Reply by ssuranaca** [reply]
It's good to see such a new look
We request you please do special effort for easy Billing and desired notification to desired person and financial year concept.

**Comment by ssuranaca** [comment]
It's good to see such a new look
We request you please do special effort for easy Billing and desired notification to desired person and financial year concept.
